cmd/network/verification.go

```diff
@@ -553,20 +552,12 @@ func (e *EgressVerification) getSecurityGroupId(ctx context.Context) (string, er
 				Name:   aws.String("tag:Name"),
 				Values: []string{fmt.Sprintf("%s-default-sg", e.cluster.ID())},
 			},
-			{
-				Name:   aws.String(fmt.Sprintf("tag:kubernetes.io/cluster/%s", e.cluster.ID())),
-				Values: []string{"owned"},
-			},
 		}
 	default:
 		filters = []types.Filter{
 			{
 				Name:   aws.String("tag:Name"),
-				Values: []string{fmt.Sprintf("%s-master-sg", e.cluster.InfraID())},
-			},
-			{
-				Name:   aws.String(fmt.Sprintf("tag:kubernetes.io/cluster/%s", e.cluster.InfraID())),
-				Values: []string{"owned"},
+				Values: []string{fmt.Sprintf("%s-master-sg", e.cluster.InfraID()), fmt.Sprintf("%s-controlplane", e.cluster.InfraID())},
```

Comment:
in 4.16m `<infra_id>-master-sg` => `<infra_id>-controlplane`. We need to check for both now.